lib/modules/platform/github/index.md

```diff
@@ -72,6 +72,12 @@ Any tokens that do not start with `ghs_` (for example tokens from GitHub Enterpr
     The installation tokens expire after 1 hour and need to be regenerated regularly.
     Alternatively as environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
 
+<!-- prettier-ignore -->
+!!! tip "Third-party tools to regenerate installation tokens"
+    If you're self-hosting Renovate within a GitHub Actions workflow, then you can use the [`jnwng/github-app-installation-token-action`](https://github.com/jnwng/github-app-installation-token-action) or [`tibdex/github-app-token`](https://github.com/tibdex/github-app-token) actions.
+    If you use Node.js/CLI, then you can use the [`github-app-installation-token`](https://github.com/gagoar/github-app-installation-token) package/library.
```

Comment:
What's the best word here? Is it package or library, or both?

Reply:
Let's go with package